tests/Microsoft.Identity.Test.Common/Core/Mocks/TokenCacheHelper.cs

```diff
@@ -199,9 +216,37 @@ internal static class TokenCacheHelper
             }
         }
 
-        internal static (MsalAccessTokenCacheItem, MsalRefreshTokenCacheItem, MsalIdTokenCacheItem, MsalAccountCacheItem) PopulateCacheWithOneAccessToken(ITokenCacheAccessor accessor)
+        internal static IEnumerable<Tuple<MsalAccessTokenCacheItem, 
+                                          MsalRefreshTokenCacheItem, 
+                                          MsalIdTokenCacheItem, 
+                                          MsalAccountCacheItem>> PopulateCacheWithAccessTokens(ITokenCacheAccessor accessor, int tokensQuantity = 1)
+        {
+            IList<Tuple<MsalAccessTokenCacheItem, MsalRefreshTokenCacheItem, MsalIdTokenCacheItem, MsalAccountCacheItem>> tokens 
+                                        = new List<Tuple<MsalAccessTokenCacheItem, MsalRefreshTokenCacheItem, MsalIdTokenCacheItem, MsalAccountCacheItem>>();
+
+            bool randomizeClientInfo = tokensQuantity > 1 ? true : false;
+
+            for (int i = 1; i <= tokensQuantity; i++)
+            {
+                var result = PopulateCacheWithOneAccessToken(accessor, randomizeClientInfo);
+                Tuple<MsalAccessTokenCacheItem, MsalRefreshTokenCacheItem, MsalIdTokenCacheItem, MsalAccountCacheItem> token = 
+                    new Tuple<MsalAccessTokenCacheItem, 
+                              MsalRefreshTokenCacheItem, 
+                              MsalIdTokenCacheItem, 
+                              MsalAccountCacheItem>(result.Item1, result.Item2, result.Item3, result.Item4);
+
+                tokens.Add(token);
+            }
+
+            return tokens;
+        }
+
+        internal static (MsalAccessTokenCacheItem, MsalRefreshTokenCacheItem, MsalIdTokenCacheItem, MsalAccountCacheItem) PopulateCacheWithOneAccessToken(ITokenCacheAccessor accessor, bool randomizeClientInfo = false)
```

Comment:
nit: params can be named:
`internal static (MsalAccessTokenCacheItem AT, MsalRefreshTokenCacheItem RT, MsalIdTokenCacheItem IDT, MsalAccountCacheItem Acc) PopulateCacheWithOneAccessToken(ITokenCacheAccessor accessor, bool randomizeClientInfo = false)`

`MsalAccountCacheItem>(result.AT, result.RT, result.IDT, result.Acc);`